types.ts

```diff
@@ -3,7 +3,7 @@ export type Options = {
   partner_id: string;
   click_id?: string;
   origin?: string;
-  lang?: 'en' | 'fr';
+  lang?: 'en' | 'fr' | 'ja' | 'zh';
```

Comment:
So we've been adding lots of new languages recently, what do you think about changing this type to a simple string?